src/Metrics.re

```diff
@@ -1 +1,16 @@
-type t;
+type t = {
+  .
+  "Timestamp": float,
+  "Documents": int,
+  "Frames": int,
+  "JSEventListeners": int,
+  "Nodes": int,
+  "LayoutCount": int,
+  "RecalcStyleCount": int,
+  "LayoutDuration": int,
+  "RecalcStyleDuration": int,
```

Comment:
LayoutDuration and RecalcStyleDuration should be floats still. I assume they need to be sub-second timings.

Reply:
Fixed in commit e35add0